### PR TITLE
Adding how to convert to production non-auth

### PR DESCRIPTION
**Pull request by djw8605**
Also included an example systemd file to start the docker container (modeled after some we have run before)

I think this should start the discussion.

### CODE REVIEW

File: stash-cache/README.md [hunk 1]
```diff
@@ -40,6 +40,56 @@ Which should output:
...
+[Install]
+WantedBy=multi-user.target
+```
+
+This systemd file can be saved to `/etc/systemd/system/docker.stash-cache.service` and started with:
+
+```
+$ systemctl start docker.stash-cache
+```
+
```
Comment (brianhlin) [inline stash-cache/README.md]: We should also tell people to `systemctl enable docker.stash-cache`

File: stash-cache/README.md [hunk 2]
```diff
@@ -40,6 +40,56 @@ Which should output:
 MPVSDSGFDNSSKTMKDDTIPTEDYEEITKESEMGDATKITSKIDANVIEKKDTDSENNITIAQDDEKVSWLQRVVEFFE
 ```
 
+Converting to Production
+------------------------
+
+Additional configuration is needed to make XCache production.
+
+1. Add a persistant caching directory.  Add the volume argument to the run command.
```
Comment (brianhlin) [inline stash-cache/README.md]:
```suggestion
1. Add a persistant caching directory from the docker host.  Add the volume argument to the run command.
```

File: stash-cache/README.md [hunk 3]
```diff
@@ -40,6 +40,56 @@ Which should output:
...
+2. Add environment variables to set the name of the cache and the directory of the persistant cache.  It is important to remember that the directory set in the environment variable points to the directory **inside** the container.  It is later mapped to a host directory with the `--volume` option.
+
+An example final `docker run` command:
+```
+$ docker run --rm --publish <HOST PORT>:8000 \
+             --volume /srv/cache:/cache
+             --env-file=/opt/xcache/.env
+             opensciencegrid/stash-cache:development
```
Comment (brianhlin) [inline stash-cache/README.md]:
```suggestion
             opensciencegrid/stash-cache:fresh
```

File: stash-cache/README.md [hunk 4]
```diff
@@ -40,6 +40,56 @@ Which should output:
 MPVSDSGFDNSSKTMKDDTIPTEDYEEITKESEMGDATKITSKIDANVIEKKDTDSENNITIAQDDEKVSWLQRVVEFFE
 ```
 
+Converting to Production
```
Comment (brianhlin) [inline stash-cache/README.md]:
```suggestion
Readying for Production
```

File: stash-cache/README.md [hunk 5]
```diff
@@ -40,6 +40,56 @@ Which should output:
...
+
+An example final `docker run` command:
+```
+$ docker run --rm --publish <HOST PORT>:8000 \
+             --volume /srv/cache:/cache
+             --env-file=/opt/xcache/.env
+             opensciencegrid/stash-cache:development
+```
+
+And an example environment file:
```
Comment (brianhlin) [inline stash-cache/README.md]:
```suggestion
Where the environment file on the docker host, `/opt/xcache/.env`, has the following contents:
```

**Comment by djw8605** [comment]
Good suggestions.  Done.

**Pull request merged**